google-finance/src/components/Common/Footer.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import logoImage from '../../assets/img/image_logo.svg';
+import { FooterLogoImage, LineImage } from '../../assets/index';
+
+const Footer = () => {
+  return (
+    <FooterContainer>
+      <FooterHeader>
+        <img src={FooterLogoImage} alt="구글금융 메뉴" />
+        <span>도움말</span>
+        <span>의견 보내기</span>
+        <span>개인정보처리방침</span>
+      </FooterHeader>
+      <FooterSub>
+        <FooterText>google.com</FooterText>
+        <img src={LineImage} alt="글자 사이 라인" />
+        <FooterText>개인정보 보호</FooterText>
+      </FooterSub>
+    </FooterContainer>
+  );
+};
+
+const FooterContainer = styled.nav`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+
+  width: 99%;
+  height: 11.5rem;
+  padding: 0 1.5rem;
+  bottom: 0;
+
+  font-family: ${(props) => props.theme.fonts.roboto_12};
+  color: ${(props) => props.theme.colors.gray_1};
+`;
+
+const FooterText = styled.span`
+  font-family: ${(props) => props.theme.fonts.productsans_10};
+  color: ${(props) => props.theme.colors.gray_2};
+`;
+
+const FooterSub = styled.span`
+  display: flex;
+`;
+
+const FooterHeader = styled.nav`
```

Comment:
호옥시 우리 CSS 적용할때 St 객체 선언해서 그 안에서 styledcomponent 선언하기로했어가지구,
```suggestion
const St={
    FooterSub : styled.span` ... `,
    FooterHeader : styled.nav` ... `,
    ...
}
```
이렇게 수정하면 좋을거같앗

Reply:
return문 안에선
```
return (
    <St.FooterContainer>
        ...
    </St.FooterContainer>
);
```
이런식으로!